extensions/attention-stream/src/background.ts

```diff
@@ -49,6 +49,9 @@ const enableEmulatorMode = Boolean(__ENABLE_EMULATOR_MODE__);
 // The Rally-assigned Study ID.
 let studyId = "attentionStream";
 
+// The "slug" on the extension store.
+let storeId = "rally-attention-stream";
```

Comment:
@aaga one thing I've noticed is that AMO and CWS somehow generated different slugs from the same `manifest.json` - on CWS it is `rally-attention-stream`, and on AMO it is `attention-stream`.

It looks like we're not allowed to change this on CWS but we are on AMO. We *could* allow multiple slugs here, but I'd like to try to unify them for consistency's sake.

Maybe we should modify AMO and change the URL in Firestore at the same time, wdyt?

Reply:
agreed!